src/TestLogger/Core/ParsedName.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Spekt Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Spekt.TestLogger.Core
+{
+    public class ParsedName
+    {
+        public ParsedName(string namespaceName, string typeName, string methodName)
+        {
+            this.NamespaceName = namespaceName;
+            this.TypeName = typeName;
+            this.MethodName = methodName;
+        }
+
+        public string NamespaceName { get; }
```

Comment:
Minor: consider `Namespace`, `Type`, `Method` since the class has the Name suffix.